assets/twitch-player.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://player.twitch.tv/js/embed/v1.js"></script>
+    <title>Twitch Player</title>
+  </head>
+  <body>
+    <div id="twitch-embed" style="margin-left: -8px; margin-top: -8px"></div>
```

Comment:
Can the width/height be determined by setting `style="width: 100%; height: 100%;"`?

Reply:
I believe I did that yesterday, the embed was looking a bit distorted and when I tried it again just now, it ended up looking like this: 
![image](https://user-images.githubusercontent.com/33183583/164093624-2b243c1f-1ca9-4dee-b27e-14c3b68d0caf.png)

I want to achieve how it was done before, with player.twitch.tv occupying 100% of the whole page but I don't see any parameters from twitch docs that support that.

Reply:
Ohh is it because the twitch player wants an explicit width/height in js? You can compute that by doing something like

```
      var width = document.getElementById('player').offsetWidth;
      var height = document.getElementById('player').offsetHeight;
```

Reply:
Width is fixed with that but not height: 
![image](https://user-images.githubusercontent.com/33183583/164103100-7eaae57e-b44e-46f0-8061-ff590360cdfc.png)